app/views/surveys/_email_survey_template.html.erb

```diff
@@ -0,0 +1,39 @@
+<script type="text/html+template"
```

Comment:
Is this ok? Is there a better content-type I could use here that would make it more obvious?